src/components/editMeasure/measureDetails/MeasureDetailsSidebar.tsx

```diff
@@ -47,7 +47,7 @@ export default function MeasureDetailsSidebar(
 
   useEffect(() => {
     if (links) setMeasureGroups(links);
-  }, []);
+  }, [measure?.groups]);
```

Comment:
Is measure Group a dependency for this hook?
but it is never used inside the hook, so there is no impact of this dependency.

Reply:
Do we want to do that? Is it a requirement?

Reply:
Yeah, for now we are implementing it, but going further when we implement the validations like: you can't create a new group until you save previous one in that case we can just remove it